content/developer/howtos/jstraining/01_owl_framework.rst

```diff
@@ -0,0 +1,310 @@
+.. _howto/jstraining/01_owl_framework:
+
+===============================
+Chapter 1: The Owl Framework 🦉
+===============================
+
+Components are the basic building blocks of user interface in Odoo. Odoo components are made with
+the `Owl framework <https://github.com/odoo/owl>`_, which is tailor-made component system for Odoo.
+
+Let us take some time to get used to Owl itself. The exercises in this section may not be realistic,
+but their purpose is to quickly understand and practice the basics of Owl.
+
+Here is an overview of what we are going to achieve in this chapter:
+
+.. image:: 01_owl_framework/overview.png
```

Comment:
There is something weird going on with the images. None of them seem to be true png, but rather jpeg disguised as png :raised_eyebrow: 
```
find . -type f -name "*.png" | xargs -I{} file  -i {} | grep -v "image/png; charset=binary$"

./content/applications/productivity/iot/devices/measurement_tool/measurement_tool_03.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/05_custom_kanban_view/customerList.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/05_custom_kanban_view/customerPager.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/05_custom_kanban_view/customerSearch.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/05_custom_kanban_view/customerFilter.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/05_custom_kanban_view/customerData.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/01_owl_framework/todo.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/01_owl_framework/counter.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/01_owl_framework/card.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/01_owl_framework/overview.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/01_owl_framework/deletingTodo.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/01_owl_framework/todoList.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/01_owl_framework/createTodo.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/01_owl_framework/togglingTodo.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/02_odoo_web_framework/statistics.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/02_odoo_web_framework/newLayout.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/02_odoo_web_framework/navigationButtons.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/02_odoo_web_framework/overview_02.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/02_odoo_web_framework/pieChart.png: image/jpeg; charset=binary
./content/developer/howtos/jstraining/02_odoo_web_framework/misc.png: image/jpeg; charset=binary
```

Reply:
Screenshot for chapter1 done in the right format